actuator_head.md

```diff
@@ -1,19 +1,36 @@
 # Construction of the linear actuator front
 
+Within this section a small hold for needles is built. It's placed at the front of the linear actuator and therefore can be moved towards the mouse if rewards should be provided. 
+
+
+
 
 ## Preparation: 3D prints {pagestep}
 
 - Print all following parts, some need to be printed twice.
+- It's recommended to print the "rotatable needle hold end" multiple times because the drilling inside is a bit tricky.
+
+
+## Print core components {pagestep}
 
+### Upper core
 
-### Core components
 ![](models/rewards_center_01.stl){color: grey}
+
+### Lower core
+
 ![](models/rewards_center_02.stl){color: grey}
 
-### Moveable needle hold (2x)
-![](models/rewards_moveable_needle_hold.stl){color: grey}
+## Print rotatable needle hold (2x) {pagestep}
```

Comment:
is "rotable" meant, rotatable is a strange name

Reply:
https://dict.leo.org/german-english/rotatable
https://dict.leo.org/german-english/rotable
this means to have the ability to rotate, rotable is sth different